.fvm/flutter_sdk

```diff
@@ -0,0 +1 @@
+C:/Users/haris/fvm/versions/stable
```

Comment:
Add `.fvm` folder to .gitignore

Reply:
Unfortunately I saw this directory just after i pushed. I'll add it 👍🏻

Reply:
#[7584e0a](https://github.com/khankhulgun/flutter_map_arcgis/pull/14/commits/7584e0a160eeda52d652ac20368de35f55f4c47d)

updated

Reply:
Nice, I see that in this commit you've introduced changes in the Gradle wrapper script. Are those intentional? I think it's not meant to be modified.

Reply:
The gitignore used in the project is outdated. Those files shouldn't be tracked in git. Lemme update .gitignore 

https://www.toptal.com/developers/gitignore/api/flutter 

You can refer this for the latest .gitignore.

Reply:
[887fda8](https://github.com/khankhulgun/flutter_map_arcgis/pull/14/commits/887fda8111b35c10713aa5da8eed8c6514fdd4fd)

fixed gitignore

Reply:
Perfect! I've just tested your branch in my local android device with flutter 2.10 and it works. I think it's ready to be merged.